### PR TITLE
[1LP][RFR] fixed paginator is_displayed and a bunch of tests failing due to absent paginator

### PR DESCRIPTION
**Pull request by izapolsk**

### CODE REVIEW

File: cfme/tests/configure/test_visual_cloud.py [hunk 1]
```diff
@@ -105,6 +105,8 @@ def test_cloud_grid_page_per_item(appliance, request, page, value, set_grid):
     limit = appliance.user.my_settings.visual.grid_view_limit
     view = navigate_to(page, 'All', use_resetter=False)
     view.toolbar.view_selector.select('Grid View')
+    if not view.entities.paginator.is_displayed:
+        pytest.skip("This page doesn't have entities and/or paginator")
```
Comment (digitronik) [inline cfme/tests/configure/test_visual_cloud.py]: @izapolsk 
1. I am suspect about skipping test as its title pointing `page_per_item`
2. There may be changes of BZ else `uncollect` will be best option.

Comment (izapolsk) [reply cfme/tests/configure/test_visual_cloud.py]: @digitronik, sorry, I didn't get what you mean. What exactly you don't like ?

tests use paginator. if paginator is absent on some page, test fails. that's incorrect.
so, now tests are skipped if there is no paginator on page.

Comment (digitronik) [reply cfme/tests/configure/test_visual_cloud.py]: It was my bad... after checking failure `screenshot` i understand that  paginator not available with `0` entities.

**Comment by mshriver** [comment]
PRT has some tests failing due to assertion failures, but not due to any changes here. Pagination checks and is_displayed update look good, thanks @izapolsk

**Pull request merged**